runtime/optwin.vim

```diff
@@ -976,9 +976,9 @@ call append("$", "updatecount\tnumber of characters typed to cause a swap file u
 call append("$", " \tset uc=" . &uc)
 call append("$", "updatetime\ttime in msec after which the swap file will be updated")
 call append("$", " \tset ut=" . &ut)
-call append("$", "maxmem\tmaximum amount of memory in Kbyte used for one buffer")
+call append("$", "maxmem\tmaximum amount of memory in Kbyte used for one buffer (ignored in Neovim)")
 call append("$", " \tset mm=" . &mm)
-call append("$", "maxmemtot\tmaximum amount of memory in Kbyte used for all buffers")
+call append("$", "maxmemtot\tmaximum amount of memory in Kbyte used for all buffers (ignored in Neovim)")
```

Comment:
These lines should be removed totally. That's what we have done for previous "hidden options".